earthfile2llb/converter.go

```diff
@@ -1773,10 +1779,28 @@ func (c *Converter) internalRun(ctx context.Context, opts ConvertRunOpts) (pllb.
 	}
 }
 
+// secretID returns query parameter style string that contains the secret
+// version, name, org, and project. The version value informs the secret
+// providers how to handle the secret name and whether to use the new
+// project-based secrets endpoints.
+func (c *Converter) secretID(name string) string {
+	v := url.Values{}
+	v.Set("name", name)
+	if c.ftrs.UseProjectSecrets {
+		v.Set("v", "1")
+		v.Set("org", c.varCollection.Org())
+		v.Set("project", c.varCollection.Project())
+	} else {
+		v.Set("v", "0")
+	}
+	return v.Encode()
+}
+
 func (c *Converter) parseSecretFlag(secretKeyValue string) (secretID string, envVar string, err error) {
 	parts := strings.SplitN(secretKeyValue, "=", 2)
 	if len(parts) == 2 {
 		if strings.HasPrefix(parts[1], "+secrets/") {
+			c.opt.Console.Printf("Deprecation: the '+secrets/' prefix is not required and support for it will be removed in an upcoming release")
```

Comment:
can you update the CHANGELOG to mention that the prefix is now optional and will eventually result in an error in a future release

Reply:
Done.